docs/uds-package-practices.md

```diff
@@ -74,3 +74,9 @@ To help maintain a UDS Package, it:
 - **Must** have a dependency management bot (such as renovate) configured to open PRs to update core package and support dependencies.
 
 - **Must** release its package to the `ghcr.io/defenseunicorns/packages/<group>` namespace as the application's name (i.e. `ghcr.io/defenseunicorns/packages/uds/mattermost`)
+
+## General
```

Comment:
"General" sounds kind of weird to me particularly at the bottom (vs `Additional` or something at the end) and with only one item, but to be honest it fine. 🤷

Reply:
Yeah I put General since it was "overarching" package guidance...